packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -9,10 +9,14 @@ import { StatefulGraphChatListClient, GraphChatListClient } from '../../stateful
 import { ChatListHeader } from '../ChatListHeader/ChatListHeader';
 import { IChatListMenuItemsProps } from '../ChatListHeader/EllipsisMenu';
 import { ChatListButtonItem } from '../ChatListHeader/ChatListButtonItem';
-import ChatListMenuItem from '../ChatListHeader/ChatListMenuItem';
+import { ChatListMenuItem } from '../ChatListHeader/ChatListMenuItem';
+import { LastReadCache } from '../../statefulClient/Caching/LastReadCache';
 
-export interface IChatListItemInteractionProps {
+export interface IChatListItemProps {
   onSelected: (e: GraphChat) => void;
+  buttonItems?: ChatListButtonItem[];
+  chatThreadsPerPage: number;
+  lastReadTimeInterval?: number;
```

Comment:
why does it need to be nullable?

Reply:
I figure 99% of people won't care to set it, so the default can just work.